libc/newhdrgen/class_implementation/classes/function.py

```diff
@@ -11,14 +11,14 @@
 
 class Function:
     def __init__(
-        self, standards, return_type, name, arguments, guard=None, attributes=[]
+        self, return_type, name, arguments, standards, guard=None, attributes=[]
```

Comment:
in future, it's better to split unrelated changes into two separate PRs. The rule of thumb is "if you need an 'and' in your commit title it's probably better as two commits"